### PR TITLE
feat(fact-figures): Adding EU styles, revamping EC - FRONT-2169

### PR DESCRIPTION
**Pull request by papegaill**
Previews:

- EC: https://ecl-preview-2041--europa-component-library.netlify.app/playground/ec/?path=/story/components-fact-figures--columns-3
- EU: https://ecl-preview-2041--europa-component-library.netlify.app/playground/eu/?path=/story/components-fact-figures--columns-3

Specs: 

- EC: https://share.goabstract.com/c83619ca-ceb4-4a43-8e05-6b4684629fed
- EU: https://share.goabstract.com/2ae34360-af9e-4833-b599-9907f5210ee8

### CODE REVIEW

File: src/implementations/twig/components/fact-figures/fact-figures.story.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const viewAll = { ...data3Col.view_all };
 const getArgTypes = (data) => {
   return {
     icon: {
-      defaultValue: data.items[0].icon.name,
+      defaultValue: system === 'ec' ? 'digital' : data.items[0].icon.name,
```
Comment (emeryro) [inline src/implementations/twig/components/fact-figures/fact-figures.story.js]: why do we force the use of this specific icon?

File: src/implementations/vanilla/components/fact-figures/fact-figures-ec.scss [hunk 2]
```diff
@@ -14,20 +14,16 @@
 }
```
Comment (emeryro) [inline src/implementations/vanilla/components/fact-figures/fact-figures-ec.scss]: on the mockups there is no background color on EC

**Review by emeryro — CHANGES_REQUESTED** [review]
For the scss files, we should use the same structure as other components:

- _fact-figures.scss (common style and variables)
- fact-figures-ec.scss (implementation of variables for ec)
- fact-figures-eu.scss (implementation of variables for ec)

File: src/implementations/vanilla/components/fact-figures/fact-figures-ec.scss (hunk 2 above)
Comment (papegaill) [reply src/implementations/vanilla/components/fact-figures/fact-figures-ec.scss]: Missing on the mockups, designers 'll be updated.

File: src/implementations/twig/components/fact-figures/fact-figures.story.js (hunk 1 above)
Comment (papegaill) [reply src/implementations/twig/components/fact-figures/fact-figures.story.js]: On EC version, they are using the same `digital` icon everywhere, on EU they are using a different icon for each items.

Comment (emeryro) [reply src/implementations/twig/components/fact-figures/fact-figures.story.js]: I don't think we need to keep this difference, that's just an example.
We could use the same content for EC and EU (including the icons)

**Review by emeryro — APPROVED** [review]

**Pull request merged**